src/components/BottomSheet/BottomSheetMenu.js

```diff
@@ -40,7 +40,7 @@ const styles = StyleSheet.create({
     height: dimen.normalizeDimen(height)
   }),
   rbsheetDraggableIcon: {
-    backgroundColor: COLORS.greySubtile1,
+    backgroundColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color constant has been updated from `COLORS.greySubtile1` to `COLORS.gray200`. Ensure that this new color constant is defined in the `COLORS` object and it matches the intended color scheme for the UI.
```diff
-    backgroundColor: COLORS.greySubtile1,
+    backgroundColor: COLORS.gray200,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->